provider/local/system_linux.go

```diff
@@ -60,7 +62,7 @@ func dnsInstallNetworkManager(name string) error {
 func dnsInstallResolved(name string) error {
 	data := []byte(fmt.Sprintf("[Resolve]\nDNS=127.0.0.1:5453\nDomains=~%s", name))
```

Comment:
Should this line use the Cluster IP rather than localhost?

Reply:
Yes, you're right. Not sure how I missed that.